### PR TITLE
feat(AsideHeader, Logo)!: make actions accessible with keyboard

### PR DESCRIPTION
**Pull request by kmenshov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

File: src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -59,40 +68,13 @@ export const Logo: React.FC<
...
-                        href={href ?? '/'}
-                        target={target}
-                        rel={target === '_self' ? undefined : 'noreferrer'}
-                        className={b('logo-link')}
-                        onClick={onClick}
-                    >
-                        {logo}
-                    </a>
-                ))}
+            {hasWrapper ? wrapper(button, Boolean(compact)) : button}
```
Comment (DarkGenius) [inline src/components/Logo/Logo.tsx:77]: The button is not suitable for all cases. When clicking the button, unnecessary visual effects occur. In addition, there may be uses of wrappers that look at the node markup. In the major version, we plan to improve this component

**Review by DarkGenius — CHANGES_REQUESTED** [review]

File: src/components/CompositeBar/Item/Item.scss [hunk 2]
```diff
@@ -103,6 +116,18 @@ $block: '.#{variables.$ns}composite-bar-item';
         height: 100%;
         cursor: pointer;
 
+        background: none;
```
Comment (DarkGenius) [inline src/components/CompositeBar/Item/Item.scss:119]: Can we move the duplicate code into a mixin?

Comment (kmenshov) [reply src/components/CompositeBar/Item/Item.scss]: Great catch, thank you! Fixed.

File: src/components/Logo/Logo.tsx (hunk 1 above)
Comment (kmenshov) [reply src/components/Logo/Logo.tsx:75]: So, I've replaced the Button element with a native <a> tag. This removes the clicking visual effects and still keeps the element tabable.

This time, however, I had to retake a couple of screenshots (Footer and MobileFooter). There seems to be no visual difference in Storybook, including Safari, but there is a slight logo shift in Playwright screenshots for some reason.

Before: https://preview.gravity-ui.com/navigation/?path=/story/components-mobilefooter--showcase

After: https://preview.gravity-ui.com/navigation/305/?path=/story/components-mobilefooter--showcase

File: src/components/Logo/Logo.tsx [hunk 3]
```diff
@@ -10,12 +10,13 @@ import './Logo.scss';
 const b = block('logo');
 
 export const Logo: React.FC<
-    LogoProps & {compact?: boolean; buttonClassName?: string; buttonWrapperClassName?: string}
```
Comment (DarkGenius) [inline src/components/Logo/Logo.tsx:13]: This is breaking change. Please retarget your PR to branch `v3`. Also message for commits with breaking changes should be starts with `feat!`

**Review by DarkGenius — APPROVED** [review]

**Pull request merged**